crates/types/src/config/bifrost.rs

```diff
@@ -318,6 +320,20 @@ pub struct ReplicatedLogletOptions {
     pub default_nodeset_size: NodeSetSize,
 }
 
+impl ReplicatedLogletOptions {
+    fn apply_common(&mut self, common: &CommonOptions) {
+        if self.default_log_replication.is_none() {
+            self.default_log_replication = Some(common.default_replication.clone());
+        }
+    }
+
+    pub fn default_log_replication(&self) -> ReplicationProperty {
+        self.default_log_replication
+            .clone()
+            .unwrap_or_else(default_log_replication)
+    }
+}
```

Comment:
This got me thinking if we should remote this option from bifrost and admin altogether. My reasoning here is to simplify the configuration surface area. Any tuning of individual replication properties can be done directly with restatectl, and perhaps keeping this ability in the config only adds to the confusion.

@tillrohrmann I'd love to hear your thoughts as well.

Reply:
I’m fine with the general approach, but here’s my take: The scheduler currently doesn’t support location-aware replication [here](https://github.com/restatedev/restate/blob/2ea30aa5b2adca66407903e94b0150b044357e9d/crates/admin/src/cluster_controller/scheduler.rs#L299), whereas Bifrost does. As a result, using a single default_replication setting for both may not provide enough flexibility for the operator to configure the initial cluster setup.

Reply:
I do like the idea of simplifying the configuration surface as there are already quite a few configuration options users need to understand. The one area that I see where having dedicated configuration options for the log and partition replication might be useful are automated deployments where one would like to avoid running an additional `restatectl` which might fail. With Flink that was sometimes what users complained about. But in case this should ever be requested, we could add it back.

If we remove the separate configuration options, then I guess the scheduler should not fail but ignore the larger location scopes.